starboard/nplb/BUILD.gn

```diff
@@ -240,36 +216,77 @@ test("nplb") {
     "ui_navigation_test.cc",
     "undefined_behavior_test.cc",
     "url_player_create_test.cc",
+    "window_create_test.cc",
+    "window_destroy_test.cc",
+    "window_get_diagonal_size_in_inches_test.cc",
+    "window_get_platform_handle_test.cc",
+    "window_get_size_test.cc",
   ]
 
-  if (!is_androidtv) {
+  media_sources = [
```

Comment:
I'd prefer this just being `sources` and the `generic_sources` being a
```
sources += [
 "a.cc",
 "b.cc",
]
```
block in the conditional statements below

I see the benefits of having it named instead though, so if you prefer that then just rename the variable to something like starboard_platform_sources as generic_sources has the implications that it would be used generically, i.e. across all platforms